shared/actions/mapActions.js

```diff
@@ -19,27 +20,26 @@ export const receiveItems = (json) => {
 };
 
 export const getMapItems = (zoomLevel, lat, long) => {
-  return (dispatch) => {
+  return async (dispatch) => {
     const queries = [];
     queries.push(createQueryObject('lat', lat));
     queries.push(createQueryObject('lng', long));
     queries.push(createQueryObject('zoom', zoomLevel));
     queries.push(createQueryObject('isThumbnail', true));
-    // todo: recover this when aws is ready: const address = `${HTTP}${SERVER_ADDR}${ENDPOINT_ITEM}${queryBuilder(queries)}`;
-    // const address = `https://goober.herokuapp.com/api/items${queryBuilder(queries)}`;
-    getAccessToken().then((accessToken) => {
+    return await getAccessToken().then(async (accessToken) => {

```

Comment:
are you sure? could you have test on it if it still works as async function when fetching the items?